egs2/edacc/asr1/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+
+test_set="test test_sub"
+train_set="dev_train"
+valid_set="dev_non_train"
+nbpe=3884 # 3884 vocabulary size of bpe could cover all the sentence in the edacc dataset
```

Comment:
The number seems weird to me. Could you please elaborate a bit on why 3884 is selected?

Reply:
Because it seems like in stage 5, when trying to use 5000 vocab size, it indicates that the dataset does not have sufficient diversity to support a vocabulary size of 5000, and the maximum feasible vocabulary size for this dataset is 3884.

Reply:
usually, we use a smaller vocab size as the bpe is for word pieces. 

If you are using 3884, it actually returns to a word-based model (this could be one major cause of the poor WER you got from the model, as the word is very sparse with a small dataset).

Given that the word size is small, I would suggest you to go with a smaller vocab size (e.g., 500 or even 100). I expect it could make a significant improvement to the model's performance.